ipmof/crystal.py

```diff
@@ -4,27 +4,38 @@
 import math
 import os
 from ipmof.forcefield import get_ff_parameters
+import ipmof.io as io
 
 
 class MOF:
     """
     MOF class that holds coordinate, atom name, and unit cell information
     """
-    def __init__(self, mof_file, file_format='mol2'):
+    def __init__(self, file_path, file_format='cif', reader='ase'):
         """
         Initialize MOF name, unit cell volume and parameters, atom names and coordinates, and
-        unique atom names and coordinates by reading values from mol2 file.
+        unique atom names and coordinates.
+        reader -> ('babel' / 'ase')
+        file_format -> ('cif' / 'dict' / 'mol2' / ...)
```

Comment:
add comment to note that file_format is only respected if 'dict'. Otherwise it just pulls from the extension.